### PR TITLE
Create Azure CLI w/ terraform image

### PR DESCRIPTION
**Pull request by Mallear**
Create an image for Azure deployment with Azure CLI and Terraform

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-docker_hub_namespace: &docker_hub_namespace ekino/docker-buildbox
+docker_hub_namespace: &docker_hub_namespace mallear/docker-buildbox
```
Comment (clemstoquart) [inline config.yml]: I think `ekino` was better here :)

Comment (Mallear) [reply config.yml]: Oops, fixed :)

Comment (clemstoquart) [reply config.yml]: Thanks.

File: azure/dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+RUN apk update \
+    && apk add --update -q --no-cache bash ca-certificates unzip wget \
+    && pip install --upgrade pip awscli \
+    && wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip -O /tmp/terraform.zip \
+    && cd /tmp \
+    && unzip -o terraform.zip \
+    && cd - \
+    && mv /tmp/terraform /usr/bin/ \
+    && chmod +x /usr/bin/terraform \
+    && apk add build-base libffi-dev openssl-dev \
```
Comment (mremi) [inline azure/dockerfile]: maybe you could group all packages installations

Comment (Mallear) [reply azure/dockerfile]: done for both apk & pip :)

File: azure/dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
+FROM python:3.6-alpine
```
[the remark this answers is not in the document]
Comment (Mallear) [reply azure/dockerfile]: Updated

**Pull request merged**